### ISSUE TITLE
sendTokenToIntercom is undefined in latest version 2.0.3

### ISSUE THREAD

**Issue by masashi-sutou** [deepak786/intercom_flutter#28]
## description

`sendTokenToIntercom` is undefined in the latest version. 

I think this commits(1f7edf4dc52e6db07b39074b9157c1b2400b987d) is not included in the latest version.

Please release the package again.

## workaround

```
intercom_flutter:
  git:
    url: git://github.com/v3rm0n/intercom_flutter
    ref: master
```

**Comment by Spikes042** [comment]
It was never released in the first place.
Pull request: #27 
Changelog: https://pub.dev/packages/intercom_flutter#-changelog-tab-

You can watch the repository to know when it's released

**Comment by masashi-sutou** [comment]
@Spikes042 
Thank you for your comment.

OK. Wait for it to be released. However, it is strange to have a README on how to set up a push notification(FCM), even if it has never been released.

**Comment by v3rm0n** [comment]
Released 2.0.4. Unfortunately it's not set up to release automatically when something changes, yet.
